data/styles/banner.css

```diff
@@ -28,25 +28,6 @@
     -gtk-icon-shadow: none;
 }
 
-.banner.card {
-    border: 1px solid shade(@banner_bg_color, 0.8);
-    box-shadow:
-        inset 0 0 0 1px alpha(shade(@banner_bg_color, 1.7), 0.05),
-        inset 0 1px 0 0 alpha(shade(@banner_bg_color, 1.7), 0.45),
-        inset 0 -1px 0 0 alpha(shade(@banner_bg_color, 1.7), 0.15),
-        0 3px 2px -1px alpha(shade(@banner_bg_color, 0.5), 0.2),
-        0 3px 5px alpha(shade(@banner_bg_color, 0.5), 0.15);
-}
-
-.banner.card:hover {
-    box-shadow:
-        inset 0 0 0 1px alpha(shade(@banner_bg_color, 1.7), 0.05),
-        inset 0 1px 0 0 alpha(shade(@banner_bg_color, 1.7), 0.45),
-        inset 0 -1px 0 0 alpha(shade(@banner_bg_color, 1.7), 0.15),
-        0 10px 8px -11px alpha(shade(@banner_bg_color, 0.6), 0.8),
-        0 8px 12px alpha(shade(@banner_bg_color, 0.8), 0.6);
-}
-
```

Comment:
Why did we get rid of this stuff? It was a pretty cool hover effect imo

Reply:
Yeah fair. I was seeing if I could get away with using default button styles (and much simpler CSS without using variables) but I can bring the glow back.